aten/src/ATen/native/cuda/ScatterGatherKernel.cu

```diff
@@ -109,8 +109,8 @@ struct _cuda_scatter_gather_internal_kernel {
     auto offset_calc = make_offset_calculator<3>(iter);
     auto loop = [=]C10_DEVICE(int i) {
       auto offsets = offset_calc.get(i);
-
-      int64_t idx_dim = *(int64_t*)(index_ptr + offsets[2]);
+      int idx_dim = *(int*)(index_ptr + offsets[2]);
+      printf("idx_dim: %d index_size: %ld\n", idx_dim, index_size);
```

Comment:
@v0dro Is this a debug statement? Can you please remove or comment this out as it is producing a lot of output in the build logs?

Reply:
Yes I'm testing one of the CI builds since it shows a garbage value for one of the index values.